mentoring/mentoring.py

```diff
@@ -148,6 +173,25 @@ def assessment_message(self):
         else:
             return None
 
+    def show_extended_feedback(self):
+        return self.extended_feedback and self.max_attempts_reached
+
+    def feedback_dispatch(self, target_json, stringify):
```

Comment:
@Kelketek nit: if I understand right, `target_json` is not a json yet, but a list(?). Would be great if it have a better name.